packages/patternfly-4/react-integration/README.md

```diff
@@ -1,7 +1,33 @@
-# react-integration
+# Patternfly 4 React Integration Testing
+
+## Overview
+Here you will find a demo application written in typescript, along with cypress integration testing.  This is used to allow developers to verify their code against a working react app that is running their latest PF4 react changes.
+
+New componets should be written in typescript and added to the demo app.  Contributors should write cypress tests to test the functionality of their components.  Contributors should also test all of the user facing interfaces and components that they are creating.
+
+## Building your react component
+After completing the development of their component a contributor should perform a build so they can start writing integration tests.
 
 1. Make your changes in packages/patternfly-4/react-core
-2. `yarn build` in packages/patternfly-4/react-core
-3. `yarn integration:setup` in packages/patternfly-4/react-integration
-4. Update the demo in packages/patternfly-4/react-integration/demo-app-ts/src/App.tsx
-5. `yarn build:demo` in packages/patternfly-4/react-integration/demo-app-ts
+2. Type `yarn build` to build your component.
+
+## Running the demo app
+To launch the demo app a contributor can run the following command:
+1. `yarn start demo:app` this will launch the demo app and watch for changes in it.
+
+## Building a typescript demo
```

Comment:
Building a TypeScript demo
Before you can start writing a Cypress test, you need to build a demo page to test against. 

1. Find demos in the component demo section and create a new folder to add your demo page. 
2. To see your demo in app navigation, add your demo to the Demo.ts file.

Reply:
Updated this but I changed step 1 too:

Go to the component demo section and create a new folder to add your demo page.